app/controllers/categoryControllers.js

```diff
@@ -0,0 +1,13 @@
+const { getCategoriesModel } = require("../models/categoryModels");
+
+const getCategoriesController = (request, response, next) => {
```

Comment:
The naming for this isn't amazing, I would probably stick with the convention we've learned so far - e.g. fetchAllCategories (in the model) & getAllCategories (in the controller)
They are a bit more clear and it can get rather confusing like this for people reading the code :)